### DISCUSSION TITLE
better swagger support

### DISCUSSION THREAD

**Discussion by tchivs** [elsa-workflows/elsa-core#3314] [Ideas]
When Elsa is referenced in an existing project, all Elsa APIs will be loaded in the default swagger page, and we have defined multiple HttpEndpoints in the process to only display triggers (route parameters are not displayed).
My idea is to group, and the process definition aggregates multiple HttpEndpoints according to the WorkflowDefinition name. I have completed these steps, can I submit a PR to you.

![image](https://user-images.githubusercontent.com/20809329/192671591-0a474aa2-7028-438a-ad90-5e86ae5cba5b.png)

![image](https://user-images.githubusercontent.com/20809329/192671695-ea9ad135-456c-40f8-972a-aed4c4513117.png)

![image](https://user-images.githubusercontent.com/20809329/192671916-f3275fb3-27dc-4333-b7d9-6c6b479cc31a.png)
![image](https://user-images.githubusercontent.com/20809329/192671962-26976286-4503-4d1e-af3f-7aca3c8edb66.png)

**Comment by sfmskywalker** [comment]
I like it! Yes, please submit a PR.
